tests/settings.yaml

```diff
@@ -1,24 +1,42 @@
 locations:
   input-dir: in/
   output-dir: out/
-  genome-dir: genome/
+  genome-fasta: genome/sample.fasta
 
 general:
   assembly: hg19
+  use_bwameth: True
+  # bismark is gold standard and uses sensible defaults
+  use_bismark: True
```

Comment:
Same comment as above about using "aligner:" instead.

Reply:
I would leave that open until we have #160 .